packages/nodes-base/nodes/SyncroMSP/v1/actions/rmm/getAll/description.ts

```diff
@@ -0,0 +1,83 @@
+import {
+	RmmProperties,
+} from '../../Interfaces';
+
+export const rmmGetAllDescription: RmmProperties = [
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		displayOptions: {
+			show: {
+				resource: [
+					'rmm',
+				],
+				operation: [
+					'getAll',
+				],
+			},
+		},
+		default: false,
+		noDataExpression: true,
+		description: 'If all results should be returned or only up to a given limit',
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		displayOptions: {
+			show: {
+				resource: [
+					'rmm',
+				],
+				operation: [
+					'getAll',
+				],
+				returnAll: [
+					false,
+				],
+			},
+		},
+		default: 25,
+		description: 'Limit the number of rows returned',
+	},
+	{
+		displayName: 'Filters',
+		name: 'additionalFilters',
```

Comment:
Why is it called "additionalFilters" and not just "filters"? Makes it sound like there are some other filters somewhere else already.